torch/functional.py

```diff
@@ -330,9 +330,38 @@ def meshgrid(*tensors: Union[Tensor, List[Tensor]]) -> Tuple[Tensor, ...]:
         return _meshgrid(*tensors)
 else:
     def meshgrid(*tensors):
-        r"""Take :math:`N` tensors, each of which can be either scalar or 1-dimensional
-        vector, and create :math:`N` N-dimensional grids, where the :math:`i` :sup:`th` grid is defined by
-        expanding the :math:`i` :sup:`th` input over dimensions defined by other inputs.
+        r"""Creates grids of coordinates specified by the 1D input tensors.
+
+        This is helpful when you want to visualize data over some
+        range of inputs. See below for plotting example.
```

Comment:
"for plotting example" => "for a plotting example"

Reply:
done